server/api/src/rsGetMiscSvrInfo.cpp

```diff
@@ -59,5 +63,42 @@ rsGetMiscSvrInfo( rsComm_t *rsComm, miscSvrInfo_t **outSvrInfo ) {
         myOutSvrInfo->serverBootTime = atoi( tmpStr );
     }
 
+    nlohmann::json certinfo_json;
+    if(irods::CS_NEG_USE_SSL == rsComm->negotiation_results && rsComm->ssl_ctx) {
+        certinfo_json["ssl_enabled"] = true;
+
+        const auto* cert = SSL_CTX_get0_certificate(rsComm->ssl_ctx);
+        std::tm tmp_tm;
+        char timebuf[40] = { 0 };
```

Comment:
This could be rewritten as:
```cpp
char timebuf[40]{};
```
Or ...
```cpp
#include <array>
...
std::array<char, 40> timebuf{};
// This gives you the following:
// - iterators
// - .size()
// - .data() for C API compatibility
// - safety checks in debug mode
```